### PR TITLE
sql-tasks peter.shumskiy

### PR DESCRIPTION
**Pull request by petr-shumskiy**
https://travis-ci.org/github/petr-shumskiy/db-assignments/builds/733798294

### CODE REVIEW

File: task/sql-tasks.js [hunk 1]
```diff
@@ -67,7 +91,26 @@ async function task_1_3(db) {
...
+           count(DISTINCT OrderID) AS 'Total number of Orders',
+           ROUND(count(distinct OrderID) / (
+               SELECT
+                   count(distinct OrderID)
+               FROM
+                   OrderDetails
+           ) * 100, 5) AS '% of all orders'
+       FROM
+           Orders
+           JOIN OrderDetails USING(OrderID)
```
Comment (vramaniuk) [inline task/sql-tasks.js]: We don't need to  join OrderDetails table because Orders includes all needed information.

File: task/sql-tasks.js [hunk 2]
```diff
@@ -177,7 +324,21 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            count(*) AS TotalOfCurrentProducts,
+            (
+                SELECT
+                    count(*)
```
Comment (vramaniuk) [inline task/sql-tasks.js]: Please, try to avoid using *, where is possible
Just count(ProductID)

File: task/sql-tasks.js [hunk 3]
```diff
@@ -199,7 +370,34 @@ async function task_1_14(db) {
...
+                    count(DISTINCT OrderID) AS total,
+                    MONTHNAME(OrderDate) AS monthName
+                FROM
+                    Orders
+                WHERE
+                    YEAR(OrderDate) = 1997
+                GROUP BY
+                    2
+            ) query
+    `)
```
Comment (vramaniuk) [inline task/sql-tasks.js]: Please, rework query with `count (if (condition))`
And this task can be solved with just one SELECT

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by IlyaMokin** [comment]
Accepted, thanks for the efforts

**Pull request closed without merging**